### DISCUSSION TITLE
[UI changed] Is there any plan to update components to material3

### DISCUSSION THREAD

**Discussion by laraib07** [Docile-Alligator/Infinity-For-Reddit#1203] [Completed Ideas] [answered]
The app is still using material component. Though it have material you monet feature, but still it looks out of place sometimes. I think the app needs a more modern and cleaner look. Maybe it should follow latest material you guidelines.

**Comment by Docile-Alligator** [comment, marked as answer]
Material Design 3 Switch will be available in the next update.
https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/071ce78b7cb6e1249e56faaef6f05fa311b83a1c
